tsconfig.json

```diff
@@ -4,9 +4,11 @@
     "jsx": "react",
     "types": ["node"],
     "outDir": "dist",
-    "target": "ES6",
+    "target": "ESNext",
```

Comment:
aegir's config has a `es2020` target, setting it to `ESNext` is a bit yolo - you'll get whatever the latest version the current tsc supports is.

This is `es2022` at the moment, are there features in that which are required?